stdlib/jdbc/src/main/ballerina/src/java.jdbc/client.bal

```diff
@@ -64,6 +78,11 @@ public type Client client object {
     #                         is executed. If the underline JDBC driver does not support it, it will return an `Error`.
     # + return - `UpdateResult` with the updated row count and key column values,
     #             else `Error` will be returned if there is an error
+    #
+    # # Deprecated
+    # New experimental jdbc2 module is introduced in 1.2 release and it will be fully supported with in 1.3 release
```

Comment:
```suggestion
    # A new experimental JDBC2 module is introduced in the Ballerina 1.2 release and it will be fully supported by the 1.3 release.
```